client/src/components/Card/Card.css

```diff
@@ -12,57 +12,65 @@
   font-family: var(--default-font);
   min-height: 9.5rem;
 }
+
 .Card--purple {
   background-color: var(--purple);
 }
+
 .Card--pink {
   background-color: var(--pink);
 }
+
 .Card--red {
   background-color: var(--red);
 }
+
 .Card--pink {
   background-color: var(--pink);
 }
+
 .Card--yellow {
   background-color: var(--yellow);
 }
+
 .Card--large {
   width: 32vw;
   max-width: 500px;
   min-width: 200px;
   max-height: 275px;
 }
+
 .Card--medium {
   width: 14.2vw;
   max-width: 225px;
   max-height: 150px;
   min-width: 180px;
-  margin: 0 0.1vw 0 0.1vw;
+  margin: 0 0.2vw 0 0.2vw;
```

Comment:
Can you use rem instead of vw?

Reply:
vw is good when something needs to be sized relative to the window (like I want the header to always take up 30% of the screen). Otherwise, it's always better to use rem